src/o365/onedrive/commands.ts

```diff
@@ -0,0 +1,5 @@
+const prefix: string = 'onedrive';
+
+export default {
+  REPORT_ONEDRIVEACTIVITYFILECOUNTS: `${prefix} report activityfilecounts`
```

Comment:
Following the command name, the cost should be named `REPORT_ACTIVITYFILECOUNTS`